worker/src/main/scala/org/apache/celeborn/service/deploy/worker/WorkerSource.scala

```diff
@@ -72,7 +73,10 @@ object WorkerSource {
   val RESERVE_SLOTS_TIME = "ReserveSlotsTime"
   val FLUSH_DATA_TIME = "FlushDataTime"
   val OPEN_STREAM_TIME = "OpenStreamTime"
+
+  // fetch data
   val FETCH_CHUNK_TIME = "FetchChunkTime"
+  val FETCH_DATA_FAIL_COUNT = "FetchDataFailCount"
```

Comment:
Can be renamed to fetchChunkFailCount. This is more clear because flink might have fetch failure but will not increment this count.